b/behat_minkbrowserkitdriver/behat_mink-browserkit-driver_v1.3.3_v1.3.4_rhel_8.4.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : behat/mink-browserkit-driver
+# Version       : v1.3.3, v1.3.4
```

Comment:
As per line 18, build script is created for version v1.3.3. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.